src/nextjournal/clerk/builder.clj

```diff
@@ -314,14 +314,16 @@
               (str path))
             (pr-str viewer)))
     (let [{:as ret :keys [out err exit]}
-          (sh "tailwindcss"
-              "--input"  tw-input
-              "--config" tw-config
-              ;; FIXME: pass inline
-              ;;"--content" (str tw-viewer)
-              ;;"--content" (str tw-folder "/**/*.edn")
-              "--output" tw-output
-              "--minify")]
+          (try (sh "tailwindcss"
+                   "--input"  tw-input
+                   "--config" tw-config
+                   ;; FIXME: pass inline
+                   ;;"--content" (str tw-viewer)
+                   ;;"--content" (str tw-folder "/**/*.edn")
+                   "--output" tw-output
+                   "--minify")
+               (catch java.io.IOException _
+                 (throw (Exception. "Clerk could not find the `tailwindcss` executable. Please install it using `npm install -D tailwindcss` and try again."))))]
```

Comment:
The install hint in the error is actually not correct which is why I didn't merge it. We currently only rely on the standalone `tailwindcss` executable, so installing it via `npm install -D tailwindcss` won't work (we'd need to call it via npx then). Do we want to support both installation methods?

Reply:
I see, maybe relying on the standalone executable was a mistake (see also https://github.com/nextjournal/clerk/blob/d80187013d7b7b96db3d8b114b8d99f687170668/.github/workflows/main.yml#L201-L200). 

Maybe we could probe the exit code of `npx tailwindcss` and then fallback to `tailwindcss` for legacy projects?

Actually we need `npm install -D tailwindcss @tailwindcss/typography` with our config.